apps/docs/docs/state/actions/actions.mdx

```diff
@@ -247,6 +127,57 @@ export class Component {
 }
 ```
 
+## RxAngular Actions
+
+This package helps to reduce code used to create composable action streams.
+It mostly is used in combination with state management libs to handle user interaction and backend communication.
+
+### Setup
+
+The coalescing features can be used directly from the `cdk` package or indirectly through the `state` package.
+To do so, install the `cdk` package and, if needed, the packages depending on it:
+
+1. Install `@rx-angular/state`
+
+```bash
+npm i @rx-angular/state
+# or
+yarn add @rx-angular/state
+```
+
+### Basic usage in a Component
+
+One of the ways `actions` can be used is to describe user input in a central place.
+To do so we define the UI interactions a user can do e.g. if we have a search box in our UI, the events it produced could get named `searchInput`.click
+
+The types for it could look like that:
+
+````typescript
+interface UI {
+  searchInput: string;
+}
+
+In the class we can use actions like follows:
+
+```typescript
+class Component {
+  ui = actions<UI>();
```

Comment:
What about using the new rxActions function in the docs? It simplifies the usage.

Reply:
Docs updates and ready for review